src/storage/internal/data_source_repository.py

```diff
@@ -25,16 +24,17 @@ class DataSourceInformation(BaseModel):
 
 
 class DataSourceRepository:
-    def __init__(self, user: User):
+    def __init__(self, user: User, get_blueprint=None):
         self.user = user
+        self.get_blueprint = get_blueprint
 
     @staticmethod
     def validate_data_source(document: dict):
         try:
             for repo in document["repositories"].values():
                 if repo.get("name"):
                     raise BadRequestException("A repository specification may not have the 'name' key.")
-                if repo["type"] == RepositoryType.MONGO.value:
+                if repo["type"] == "mongo-db":
```

Comment:
Not sure why we removed the enum? :)

Reply:
It's the only placed its used. So didn't see much point in having it as an enum